### DISCUSSION TITLE
Dumping everything into the top directory

### DISCUSSION THREAD

**Discussion by michaeyk** [nvim-neorg/neorg#1168] [Q&A]
Is anyone doing this? I have various directories for classes, projects, home, etc.  When creating a new note, it puts it into my root. I then move the note to its correct directory and fix my links to reflect its new correct location.

I'm only doing this to make myself feel my notes are neat and tidy. In fact, the only time I'm traversing the directory tree is to put the new note in its correct place. I use telescope to search for notes or links from other notes. 

Is there some downside of using a single directory for all my notes?

**Comment by pysan3** [comment]
How about zettlekasten style of management? 

It has four top dirs, and (basically) you dump everything in one of those. It keeps all pretty tidy but you don't have to think much either.
